docs/getting_started/modin_vs_dask_vs_koalas.rst

```diff
@@ -0,0 +1,91 @@
+Modin vs. Dask DataFrame and Modin vs. Koalas
+===============================================
+
+Libraries such as Dask DataFrame (DaskDF for short) and Koalas aim to support the pandas API on top of distributed computing frameworks, Dask and Spark respectively. Instead, Modin aims to preserve the pandas API and semantics as is, while abstracting away the details of the distributed computing framework underneath. Thus, the aims of these libraries are fundamentally different.
+
+Specifically, Modin enables pandas-like
+
+* row and column-parallel operations, unlike DaskDF and Koalas that only support row-parallel operations
+* indexing & ordering semantics, unlike DaskDF and Koalas that deviate from these semantics
+* eager execution, unlike DaskDF and Koalas that provide lazy execution
+
+As a result, Modin's coverage is `more than 90% <https://github.com/modin-project/modin#pandas-api-coverage>`_ of the pandas API, while DaskDF and Koalas' coverage is about 55%. 
+
+.. figure:: ../img/api_coverage_comparison.svg
+   :align: center
+   :alt: Percentage coverage of the pandas API after deduplication
+
+For more technical details please see our VLDB 2022 paper, referenced `here <https://people.eecs.berkeley.edu/~totemtang/paper/Modin.pdf>`_. 
+
+Brief Overview of DaskDF and Koalas
+-----------------------------------
+
+Dask's `DataFrame <https://docs.dask.org/en/stable/dataframe.html>`_ (DaskDF) is effectively a meta-frame, partitioning and scheduling many smaller ``pandas.DataFrame`` objects. Users construct a task graph of dataframe computation step by step and then trigger computation using the ``compute`` function.
```

Comment:
```suggestion
Dask's `DataFrame <https://docs.dask.org/en/stable/dataframe.html>`_ (DaskDF) acts as a master dataframe, partitioning and scheduling many smaller ``pandas.DataFrame`` objects. Users construct a task graph of dataframe computation step by step and then trigger computation using the ``compute`` function.
```

Reply:
I would prefer "meta DataFrame". We should avoid the use of the term "master" due to racist connotations.